package/MDAnalysis/analysis/psa.py

```diff
@@ -254,6 +254,10 @@ def get_path_metric_func(name):
     except KeyError as key:
         print("Path metric {0} not found. Valid selections: ".format(key))
```

Comment:
Maybe change it to a value error. But I would keep the exception. An invalid selection does sound like a grave mistake